### PR TITLE
Ported SelectWidget to Fluent UI theme

### PR DESCRIPTION
**Pull request by sramaswami11**
### Reasons for making this change

[Please describe them here]

If this is related to existing tickets, include links to them as well.

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

image

### CODE REVIEW

**Review by epicfaace — APPROVED** [review]

File: packages/fluent-ui/src/SelectWidget/SelectWidget.tsx [hunk 1]
```diff
@@ -59,49 +53,35 @@ const SelectWidget = ({
...
 
-  const emptyValue = multiple ? [] : '';
+  const _onChange = ({}, value: any) => onChange(value.key);
 
-  const _onChange = ({
-    target: { value },
-  }: React.ChangeEvent<{ name?: string; value: unknown }>) =>
-    onChange(processValue(schema, value));
   const _onBlur = ({ target: { value } }: React.FocusEvent<HTMLInputElement>) =>
     onBlur(id, processValue(schema, value));
```
Comment (epicfaace) [inline packages/fluent-ui/src/SelectWidget/SelectWidget.tsx]: Please don't call processValue here, also please remove the actual definition of processValue

File: packages/fluent-ui/src/SelectWidget/SelectWidget.tsx [hunk 2]
```diff
@@ -59,49 +53,35 @@ const SelectWidget = ({
...
   const _onFocus = ({
     target: { value },
   }: React.FocusEvent<HTMLInputElement>) =>
     onFocus(id, processValue(schema, value));
 
+  const dropDown = (enumOptions as any[]) || [];
+
+  const newOptions = dropDown.map(option => ({
```
Comment (epicfaace) [inline packages/fluent-ui/src/SelectWidget/SelectWidget.tsx]: Please remove the line above and say
```suggestion
  const newOptions = enumOptions.map(option => ({
```

**Review by epicfaace — CHANGES_REQUESTED** [review]

File: packages/fluent-ui/src/SelectWidget/SelectWidget.tsx (hunk 1 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/SelectWidget/SelectWidget.tsx]: Done.

File: packages/fluent-ui/src/SelectWidget/SelectWidget.tsx (hunk 2 above)
Comment (sramaswami11) [reply packages/fluent-ui/src/SelectWidget/SelectWidget.tsx]: Done.

**Review by epicfaace — APPROVED** [review]

**Pull request merged**